src/lib/components/Accordion.react.js

```diff
@@ -56,6 +56,12 @@ Accordion.propTypes = {
      * Array of accordion children.
      */
     children: PropTypes.arrayOf(PropTypes.node),
+
+    /**
+     * Indexes on elements to jump to.
+     */
+    links: PropTypes.arrayOf(PropTypes.number),
```

Comment:
These names don't work together very well. Are they links? are they indices? Maybe more detail needed in docstring

Reply:
good point